src/components/MultipleAvatars.tsx

```diff
@@ -80,142 +72,158 @@ const avatarSizeToStylesMap = {
+    shouldStackHorizontally = false,
+    shouldDisplayAvatarsInRows = false,
+    isHovered = false,
+    isPressed = false,
+    isFocusMode = false,
+    isInReportAction = false,
+    shouldShowTooltip = true,
+    shouldUseCardBackground = false,
+    maxAvatarsInRow = CONST.AVATAR_ROW_SIZE.DEFAULT,
+}: MultipleAvatarsProps) {
+    let avatarContainerStyles = StyleUtils.getContainerStyles(size, isInReportAction);
+    const {singleAvatarStyle, secondAvatarStyles} = useMemo(() => avatarSizeToStylesMap[size as AvatarSizeToStyles] ?? avatarSizeToStylesMap.default, [size]);
+
+    const tooltipTexts = useMemo(() => (shouldShowTooltip ? icons.map((icon) => icon.name) : ['']), [shouldShowTooltip, icons]);
     const avatarSize = useMemo(() => {
-        if (props.isFocusMode) {
+        if (isFocusMode) {
             return CONST.AVATAR_SIZE.MID_SUBSCRIPT;
         }
 
-        if (props.size === CONST.AVATAR_SIZE.LARGE) {
+        if (size === CONST.AVATAR_SIZE.LARGE) {
             return CONST.AVATAR_SIZE.MEDIUM;
         }
 
         return CONST.AVATAR_SIZE.SMALLER;
-    }, [props.isFocusMode, props.size]);
+    }, [isFocusMode, size]);
 
     const avatarRows = useMemo(() => {
         // If we're not displaying avatars in rows or the number of icons is less than or equal to the max avatars in a row, return a single row
-        if (!props.shouldDisplayAvatarsInRows || props.icons.length <= props.maxAvatarsInRow) {
-            return [props.icons];
+        if (!shouldDisplayAvatarsInRows || icons.length <= maxAvatarsInRow) {
+            return [icons];
         }
 
         // Calculate the size of each row
-        const rowSize = Math.min(Math.ceil(props.icons.length / 2), props.maxAvatarsInRow);
+        const rowSize = Math.min(Math.ceil(icons.length / 2), maxAvatarsInRow);
 
         // Slice the icons array into two rows
-        const firstRow = props.icons.slice(rowSize);
-        const secondRow = props.icons.slice(0, rowSize);
+        const firstRow = icons.slice(rowSize);
+        const secondRow = icons.slice(0, rowSize);
 
         // Update the state with the two rows as an array
         return [firstRow, secondRow];
-    }, [props.icons, props.maxAvatarsInRow, props.shouldDisplayAvatarsInRows]);
+    }, [icons, maxAvatarsInRow, shouldDisplayAvatarsInRows]);
 
-    if (!props.icons.length) {
+    if (!icons.length) {
         return null;
     }
 
-    if (props.icons.length === 1 && !props.shouldStackHorizontally) {
+    if (icons.length === 1 && !shouldStackHorizontally) {
         return (
             <UserDetailsTooltip
-                accountID={props.icons[0].id}
-                icon={props.icons[0]}
+                accountID={icons[0].id}
+                icon={icons[0]}
                 fallbackUserDetails={{
-                    displayName: props.icons[0].name,
-                    avatar: props.icons[0].avatar,
+                    displayName: icons[0].name,
+                    avatar: icons[0].source,
```

Comment:
Can you explain to me why has this changed across the files? I don't understand why it was `icon.avatar` before and now it's `icon.source`, maybe I'm missing something 😄

Reply:
@BartoszGrajdek I haven't found any place where icons have an avatar field, so it looked wrong, that's why I replaced it with source which makes more sense

Reply:
Wow, this is really weird, I do not know how it's been working before then... Have you heard anything about it @blazejkustra?

![Screenshot 2023-11-08 at 14 42 28](https://github.com/Expensify/App/assets/41169160/ffc16d13-5aeb-406a-b966-472ceb276b96)

Reply:
Nooo, that has to be a mistake. It worked as we have fallback icons and probably nobody noticed 😄

Reply:
@blazejkustra Maybe it makes sense to remove it then?

Reply:
Why would you remove it? 🤔

Reply:
@blazejkustra Because the tooltip anyway[ use the icon from props first](https://github.com/Expensify/App/blob/main/src/components/UserDetailsTooltip/BaseUserDetailsTooltip.web.js#L49) and only then try to get user avatar. Since we pass the same icon as a prop, it doesn't make a difference if we pass it to the fallback details. Plus, it looks like before, `undefined` was always put as an avatar, so we shouldn't break anything.

Reply:
Thank you for explanation, I'm fine with removing!

Reply:
Updated!